earth_enterprise/src/server/wsgi/serve/basic_types.py

```diff
@@ -22,7 +22,7 @@
 from common import string_utils
 
 # Portable files extensions set.
-PORTABLE_EXTS_SET = {".glm", ".glb", ".glc"}
+PORTABLE_EXTS_SET = [".glm", ".glb", ".glc"]
```

Comment:
Very minor: This isn't a set any more, so we could change the variable name to remove `_SET`. Or, not.